frontend/public/components/ingress.jsx

```diff
@@ -12,7 +12,7 @@ const getHosts = (ingress) => {
   const hosts = ingressValidHosts(ingress);
 
   if (hosts.length) {
-    return <div>{hosts.join(', ')}</div>;
+    return <div className="co-truncate">{hosts.join(', ')}</div>;
```

Comment:
We should add a `title` attribute so you can hover and see the full value (here and below).

```suggestion
    const hostsStr = hosts.join(', ');
    return <div className="co-truncate" title={hostsStr}>{hostsStr}</div>;
```